libs/s25main/ogl/OpenGLRenderer.cpp

```diff
@@ -111,7 +112,7 @@ void OpenGLRenderer::DrawLine(DrawPoint pt1, DrawPoint pt2, unsigned width, unsi
     glDisable(GL_TEXTURE_2D);
     glColor4ub(GetRed(color), GetGreen(color), GetBlue(color), GetAlpha(color));
 
-    glLineWidth(static_cast<GLfloat>(width));
+    glLineWidth(static_cast<GLfloat>(width) * VIDEODRIVER.GetGuiScale().scaleFactor());
```

Comment:
Add a scalar overload to `screenToView()` and `viewToScreen()` and use it here. Should be clearer.